needle/core/device/remote_operations.py

```diff
@@ -43,6 +43,11 @@ def file_copy(self, src, dst):
         cmd = "cp {} {}".format(src, dst)
         self.command_blocking(cmd)
 
+    def file_read(self, fname):

```

Comment:
A function `read_file` already exist in this module. Please remove this redundant function